File: TD/TD2/Code/5.cpp
/**
 * Tester si un entier choisi par USER est multiple de 5 ou de 7
 */
#include <iostream>

int main()
{
  int a;

  std::cout << "Donne une valeur : " << std::flush;
  std::cin >> a;

  if (a % 5 == 0 && a % 7 == 0)
    std::cout << a << " est multiple de 5 et de 7" << std::endl;
  else if (a % 5 == 0)
    std::cout << a << " est multiple de 5" << std::endl;
  else if (a % 7 == 0)
    std::cout << a << " est multiple de 7" << std::endl;
  else
    std::cout << "C'est ni l'un ni l'autre" << std::endl;

  return 0;
}

File: TD/TD2/Code/Python/5.py
#!/usr/bin/python
# Tester si un entier choisi par USER est multiple de 5 ou de 7
print('Donne moi une valeur : ', end='')
a = int(input())

if (a % 5 == 0) and (a % 7 == 0):
    print('C\'est le multiple de 5 et de 7')
elif a % 5 == 0:
    print('C\'est le multiple de 5')
elif a % 7 == 0:
    print('C\'est le multiple de 7')
else:
    print('C\'est ni l\'un ni l\'autre')
